exercises/luhn/luhn_test.exs

```diff
@@ -8,37 +8,67 @@ ExUnit.configure exclude: :pending, trace: true
 defmodule LuhnTest do
   use ExUnit.Case
 
-  test "checksum" do
-    assert Luhn.checksum("4913") == 22
+  test "single digit strings can not be valid" do
+    assert Luhn.valid?("1") == false
```

Comment:
While we're in here, we don't actually need the `== false` in any of these assertions. Since `Luhn.valid/1` already returns a boolean, we can use `assert` and `refute` without comparing that boolean to a second boolean.